lib/std/secrets.nim

```diff
@@ -0,0 +1,98 @@
+import os
```

Comment:
only because of osLastError ? maybe we can refactor to avoid this large dependency?